src/containers/shared/components/Transaction/README.md

```diff
@@ -0,0 +1,16 @@
+## How Transactions Are Defined
+
+Each transaction has its properties defined in `./{TransactionName}/index.ts`.
+
+- **Description**: A React component that defines the "Description" portion of a transaction's "Detailed" tab.
+- **Simple**: A React component that defines the left column of a transaction's "Simple" tab.
+- **TableDetail**: A React component that defines the body of transaction on the account, ledger, token, or nft page.
+- **mapper**: A function which takes a transaction object and returns additional properties to map onto a transaction.
+
+  This is run when a transaction is received from the server and is useful for transforming complex properties derived
+  from nodes.
+
+This object is then provided in `./index.ts`
```

Comment:
I think it would also be useful to heavily/aggressively comment the `OfferCreate` components, since that will be the template for all the other transactions. I personally learn how to edit codebases via reading/modifying existing code, and that would be very useful for that learning style.

Reply:
I will do that when I start on the next  round.  There is probably a better one to do it in that is much simpler.